<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCallShareWidget</class>
 <widget class="QWidget" name="VideoCallShareWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: 'Microsoft YaHei';
	font-size:16px;
    background:#1d2129;
}

#lbl_info{
	font-family: 'Microsoft YaHei';
	font-size:16px;
	color:#fff;
}
#btn_close{
	border:none;
	color:#fff;
}

QScrollArea
{
border:none;
}

QScrollBar:vertical 
{
border: none;
background:#000000;
width: 4px;
margin: 0px;
}
QScrollBar::handle:vertical { 
	background:#4E5969; 
    min-height: 20px;
	width: 4px;
	border-radius:2px;
}
QScrollBar::add-line:vertical {background-color: 
rgb(0,0,0);height: 4px;}
QScrollBar::sub-line:vertical {background-color: 
rgb(0,0,0);height: 4px;}
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical 
{height: 0px;}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{background: #000000;}

QTabBar::tab{
font-style: normal;
font-weight: 400;
font-size: 15px;
line-height: 20px;
width:395px;		
height:32; 			
margin-bottom:15px;
border-bottom:1px solid #4E5969;
}

QTabBar::tab:!selected {
color: #86909C;
border-bottom:1px solid #4E5969;
}

QTabBar::tab:selected {
color: #4080FF;
border-bottom:1px solid #4080FF;
}

QTabWidget::pane{
	border:none;
}
QTabWidget::tab-bar { 
	alignment: center; 
} </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>32</number>
     </property>
     <property name="topMargin">
      <number>24</number>
     </property>
     <property name="rightMargin">
      <number>32</number>
     </property>
     <property name="bottomMargin">
      <number>24</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_info">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../resource/resource.qrc">
         <normaloff>:/img/image/close.png</normaloff>:/img/image/close.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:#1d2129;



</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QScrollArea" name="screenScrollArea">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <widget class="ShareViewContainer" name="screen_views">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>376</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QScrollArea" name="windowScrollArea">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <widget class="ShareViewContainer" name="window_views">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>800</width>
         <height>376</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShareViewContainer</class>
   <extends>QWidget</extends>
   <header>share_view_container.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
